lm-simulator/lm_simulator/schemas.py

```diff
@@ -1,4 +1,4 @@
-from typing import List, Optional
+from typing import List
```

Comment:
Since we're on Python 3.12, we should use `list` in type hints instead of `List`.